feature/permissions/src/main/java/com/google/jetpackcamera/permissions/ui/TestTags.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.jetpackcamera.permissions.ui
+
+const val REQUEST_PERMISSION_BUTTON_TAG = "RequestPermissionTag"
+const val CAMERA_PERMISSION_TAG = "CameraPermissionTag"
+const val AUDIO_RECORD_PERMISSION_TAG = "RecordAudioPermissionTag"
```

Comment:
We're trying to keep these constant names in sync with the string literals:
`REQUEST_PERMISSION_BUTTON` -> `"RequestPermissionButton"`